submission/persistence-filesystem/src/test/kotlin/ac/uk/ebi/biostd/persistence/filesystem/nfs/NfsFtpServiceTest.kt

```diff
@@ -114,12 +137,36 @@ internal class NfsFtpServiceTest(
                 assertThat(result).content().isEqualTo("file content")
                 assertThat(file).doesNotExist()
             }
-        }
+
+        @Test
+        fun `suppress submission file`() =
+            runTest {
+                val subFolder = folderResolver.getPublicSubFolder(REL_PATH).toFile()
+                val filesPath = subFolder.createDirectory(FILES_PATH)
+                val file = filesPath.createFile("move-test.txt", "move content")
+                val nfsFile =
+                    NfsFile(
+                        filePath = "move-test.txt",
+                        relPath = "Files/move-test.txt",
+                        file = file,
+                        fullPath = file.absolutePath,
+                        md5 = file.md5(),
+                        size = file.size(),
+                    )
+
+                val privateFolder = folderResolver.getPrivateSubFolder(SECRET_KEY, REL_PATH).resolve(FILES_PATH)
+                val suppressedPath = privateFolder.resolve("move-test.txt")
+                val suppressed = testInstance.unReleaseSubmissionFile(subInfo, nfsFile) as NfsFile
```

Comment:
I realize I never commended this but something that I do in test in separate the classic test part by line break (arrange, action and assert). I would suggest we addopt this. 

https://automationpanda.com/2020/07/07/arrange-act-assert-a-pattern-for-writing-good-tests/